Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master
 
 # Add the repos containing nginx and galaxy charts
 RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
-    helm repo add galaxy https://raw.githubusercontent.com/cloudve/helm-charts/terra/ && \
+    helm repo add galaxy https://raw.githubusercontent.com/rtitle/helm-charts/terra/ && \
```

Comment:
TODO: revert to not my fork once https://github.com/galaxyproject/galaxykubeman-helm/pull/29 is merged